<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HOGCv</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="setPosTrainToolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Positive Training Dir</string>
    </property>
   </widget>
   <widget class="QToolButton" name="setNegTrainToolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Negative Training Dir</string>
    </property>
   </widget>
   <widget class="QToolButton" name="setPosTestToolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Positive Testing Dir</string>
    </property>
   </widget>
   <widget class="QToolButton" name="setNegTestToolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Negative Testing Dir</string>
    </property>
   </widget>
   <widget class="QLabel" name="posTrainDirLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="negTestDirLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="posTestDirLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="negTrainDirLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="filesTabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="posTrainTab">
     <attribute name="title">
      <string>Positive Train</string>
     </attribute>
     <widget class="QListWidget" name="posTrainList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>701</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="negTrainTab">
     <attribute name="title">
      <string>Negative Train</string>
     </attribute>
     <widget class="QListWidget" name="negTrainList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>691</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="posTestTab">
     <attribute name="title">
      <string>Positive Test</string>
     </attribute>
     <widget class="QListWidget" name="posTestList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>691</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="negTestTab">
     <attribute name="title">
      <string>Negative Test</string>
     </attribute>
     <widget class="QListWidget" name="negTestList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>691</width>
        <height>351</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="displayImageButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DisplayImage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classifyButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Classify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSet_Parameters"/>
    <addaction name="actionSet_Positive_Training_Dir"/>
    <addaction name="actionSet_Negative_Training_Dir"/>
    <addaction name="actionSet_Positive_Testing_Dir"/>
    <addaction name="actionSet_Negative_Testing_Dir"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSet_Parameters">
   <property name="text">
    <string>Set Parameters</string>
   </property>
  </action>
  <action name="actionSet_Positive_Training_Dir">
   <property name="text">
    <string>Set Positive Training Dir</string>
   </property>
  </action>
  <action name="actionSet_Negative_Training_Dir">
   <property name="text">
    <string>Set Negative Training Dir</string>
   </property>
  </action>
  <action name="actionSet_Positive_Testing_Dir">
   <property name="text">
    <string>Set Positive Testing Dir</string>
   </property>
  </action>
  <action name="actionSet_Negative_Testing_Dir">
   <property name="text">
    <string>Set Negative Testing Dir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
